runtime/src/iree/vm/value.h

```diff
@@ -30,10 +30,12 @@ typedef enum iree_vm_value_type_e {
   IREE_VM_VALUE_TYPE_I32 = 3,
   // int64_t.
   IREE_VM_VALUE_TYPE_I64 = 4,
+  // halft_t.
+  IREE_VM_VALUE_TYPE_F16 = 5,
   // float.
-  IREE_VM_VALUE_TYPE_F32 = 5,
+  IREE_VM_VALUE_TYPE_F32 = 6,
   // double.
-  IREE_VM_VALUE_TYPE_F64 = 6,
+  IREE_VM_VALUE_TYPE_F64 = 7,
```

Comment:
Check with @benvanik if it's OK to shift existing enum values --- these are never serialized, right?